### PR TITLE
Exception messages have been made secure

### PR DESCRIPTION
**Pull request by masdevas**

### CODE REVIEW

**Comment by SmirnovEgorRu** [comment]
make test

File: lang_interface/java/com/intel/daal/utils/LibUtils.java [hunk 1]
```diff
@@ -80,19 +80,19 @@ private static void loadFromJar(String path, String name) throws Throwable
...
         if (fileOut == null) {
-            logger.log(logLevel, "DONE: Loading library " + FullName + " as resource.");
+            logger.log(logLevel, "DONE: Loading library as resource.");
             return;
         }
 
         InputStream streamIn = LibUtils.class.getResourceAsStream(LIBRARY_PATH_IN_JAR + "/" + FullName);
         if (streamIn == null)
         {
-            throw new IOException("Error: No resource " + LIBRARY_PATH_IN_JAR + "/" + FullName + " found.");
```
Comment (SmirnovEgorRu) [inline lang_interface/java/com/intel/daal/utils/LibUtils.java]: LIBRARY_PATH_IN_JAR is a constant, which can be viewed in sources. I'm not sure that it is a security problem. But okay.

**Comment by SmirnovEgorRu** [comment]
Here are some errors in mysql samples. Please, fix it before commit:
sources/odbc_wrapper.h(69): error: identifier "Error" is undefined
      void add(const Error &e)

**Comment by masdevas** [comment]
make test

**Pull request merged**